build/build.go

```diff
@@ -62,7 +62,14 @@ func NewBuildContext(installSuffix string, buildTags []string) *build.Context {
 // If an error occurs, Import returns a non-nil error and a nil
 // *PackageData.
 func Import(path string, mode build.ImportMode, installSuffix string, buildTags []string) (*PackageData, error) {
-	return importWithSrcDir(path, "", mode, installSuffix, buildTags)
+	wd, err := os.Getwd()
+	if err != nil {
+		// Getwd may fail if we're in GOARCH=js mode. That's okay, handle
+		// it by falling back to empty working directory. It just means
+		// Import will not be able to resolve relative import paths.
+		wd = ""
```

Comment:
This should be at the tool level. It already has https://github.com/gopherjs/gopherjs/blob/master/tool.go#L40 which you can use. Either export `importWithSrcDir` or consider renaming `importWithSrcDir` to `Import`. I wouldn't mind breaking the API in this way, it is easy to detect and fix for the users and then it would also fit better to `go/build.Context.Import`.